src/text/text1.asm

```diff
@@ -38,7 +38,7 @@ PKMNPWRText:
 	text "PKMN PWR"
 	done
 
-Text000b:
+Unused000b: ; Unused (Pokemon Card)
```

Comment:
How about "UnusedText" as a prefix instead of "Unused"?

Reply:
I had actually did this first thing on my side already. The rest I have finished and just pushed.